packages/website/ts/utils/utils.ts

```diff
@@ -394,6 +395,13 @@ export const utils = {
             : 2;
         const format = `0,0.${_.repeat('0', precision)}`;
         const formattedAmount = numeral(unitAmount).format(format);
+        if (_.isNaN(formattedAmount)) {
+            // tslint:disable-next-line:no-floating-promises
```

Comment:
side note, perhaps we should change `reportAsync` to a function called `report` that just returns void but internally fires a floating promise